src/app/common/counter/pipe/counter-price-filter.pipe.spec.ts

```diff
@@ -1,8 +1,54 @@
-import { CounterPriceFilterPipe } from './counter-price-filter.pipe';
+import { CounterPriceFilterPipe } from "./counter-price-filter.pipe";
 
-describe('CounterPriceFilterPipe', () => {
-  it('create an instance', () => {
+describe("CounterPriceFilterPipe", () => {
+  it("create an instance", () => {
     const pipe = new CounterPriceFilterPipe();
     expect(pipe).toBeTruthy();
   });
+
+  it("should select only flights with prices less than or equal to the selected maximum", () => {
+    const pipe = new CounterPriceFilterPipe(),
+      maxPrice = 100,
+      cheaperFlight = {
```

Comment:
I would consider moving this mock data to another folder. We will probably need flight data a number of times.